Makefile

```diff
@@ -9,6 +9,12 @@ build:
 test: wait_mysql wait_minio
 	docker-compose run web python manage.py test --keepdb --verbosity=2
 
+test-backend-all:
+	docker-compose run web python manage.py test --keepdb --verbosity=2
+
+test-backend-fast:
+	docker-compose run web python manage.py test --keepdb --verbosity=2 --exclude-tag=factory_many --exclude-tag=factory_simple
```

Comment:
Mam wrażenie, że tak szczegółowe tagi sprawią, że za chwilę będziemy mieli tutaj 30 pozycji i więcej, a zatem będzie trudniej tym zarządzić. Ja jak chce szybko testować to uruchamiam  ```docker-compose run web python manage.py test --keepdb --verbosity=2 small_eod.cases```, albo jeszcze bardziej specyficznie, bo można wskazać dowolną część, która zostanie przetestowana (nawet jeden test) kopiując ścieżkę z logów budowania.